### PR TITLE
Lab 7 KV: define same name for all resource´s RG to fix deleting of ALL resources (not just KV)

### PR DESCRIPTION
**Pull request by unaihuete93**
# Module: 07
## Lab/Demo: 07

Fixes # .

Changes proposed in this pull request:

- ADO template is using different RG name for WebApp and SQL, make it the same as KV one, to simplify&fix deletion of lab resources.
-
-

### CODE REVIEW

File: Instructions/Labs/AZ400_M07_Integrating_Azure_Key_Vault_with_Azure_DevOps.md [hunk 1]
```diff
@@ -208,7 +208,9 @@ This will open another browser tab displaying the **Service connections** pane i
 
     > **Note**: The **Override template parameters** content references the **sqldbpassword** variable to set the mySQL admin password. This will provision the MySQL database defined in the ARM template using the password that you have specified in the key vault. 
 
-1.  You may complete the pipeline definition by specifying the subscription (if need subscription used, click on **authorize** )and location for the task. **Repeat** the same for the last task in the pipeline **Azure App Service Deploy**. 
+1.  You may complete the pipeline definition by specifying the subscription (if need subscription used, click on **authorize** )and location for the task. **Repeat** the same for the last task in the pipeline **Azure App Service Deploy**.
+
+1.  In the variables ta, modify the **resourcegroup** variable to **az400m07l01-RG**.
```
Comment (mkennten) [inline Instructions/Labs/AZ400_M07_Integrating_Azure_Key_Vault_with_Azure_DevOps.md:213]: Hi @unaihuete93, I think you want to say "In the variables tab", correct?

File: Instructions/Labs/AZ400_M07_Integrating_Azure_Key_Vault_with_Azure_DevOps.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ In this lab, you will see how you can integrate Azure Key Vault with an Azure De
 After you complete this lab, you will be able to:
 
 -   Create an Azure Active Directory (Azure AD) service principal.
--   Create an Azure key vault. 
+-   Create an Azure key vault.
```
Comment (mkennten) [inline Instructions/Labs/AZ400_M07_Integrating_Azure_Key_Vault_with_Azure_DevOps.md:26]: According to the doc it should say "Azure Key Vault" (capitals). This needs also to be corrected at other locations in the same file (we can also change this later).

**Review by mkennten — COMMENTED** [review]
@unaihuete93 Please have a look at let me know what you think

File: Instructions/Labs/AZ400_M07_Integrating_Azure_Key_Vault_with_Azure_DevOps.md (hunk 1 above)
Comment (unaihuete93) [reply Instructions/Labs/AZ400_M07_Integrating_Azure_Key_Vault_with_Azure_DevOps.md]: fixed :)

File: Instructions/Labs/AZ400_M07_Integrating_Azure_Key_Vault_with_Azure_DevOps.md (hunk 2 above)
Comment (unaihuete93) [reply Instructions/Labs/AZ400_M07_Integrating_Azure_Key_Vault_with_Azure_DevOps.md]: fixed :)

**Comment by unaihuete93** [comment]
@mkennten thanks for your feedback comments. New commits implemented proposed changes :)

File: Instructions/Labs/AZ400_M07_Integrating_Azure_Key_Vault_with_Azure_DevOps.md (hunk 1 above)
Comment (mkennten) [reply Instructions/Labs/AZ400_M07_Integrating_Azure_Key_Vault_with_Azure_DevOps.md]: Looks good now.

**Comment by unaihuete93** [comment]
Merging is messing up with conflicts, I will create a new PR

**Pull request closed without merging**